internal/actions/sync_branch.go

```diff
@@ -420,6 +423,36 @@ func syncBranchRebase(
 	return nil, nil
 }
 
+func fetchUpstreamCommits(repo *git.Repo, tx meta.WriteTx, branch meta.Branch) (string, error) {
+	parent := branch
+	for parent.Parent.Name != "" {
+		parent, _ = tx.Branch(parent.Parent.Name)
+	}
```

Comment:
I think this could just be simplified to something like

```
trunk, ok := meta.Trunk(branch)
```

Reply:
Done.